test/e2e/#smoke.test.js

```diff
@@ -53,10 +58,12 @@ users.forEach(user => {
     await t.useRole(user.role).navigateTo(movesByDay)
   })(`As ${user.name}`, async t => {
     await t
-      .expect(page.nodes.appHeader.exists)
+      .expect(page.nodes.appHeaderOrganisation.exists)
+      .ok()
+      .expect(page.nodes.appHeaderProduct.exists)
       .ok()
       .expect(page.nodes.username.innerText)
-      .eql(user.username)
+      .eql(`${user.displayName}\nManage your details`)
```

Comment:
This could be changed to `.contains` so you don't need to worry about it including the manage link.